Runtime. 5.0.9 release notes (#1240)
astronomer/docs: astro/runtime-release-notes.md

To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).

## Astro Runtime 5.0.9

- Release date: September 20, 2022
- Airflow version: 2.3.4

### Additional improvements

> - Upgraded `astronomer-providers` to 1.8.1, which includes various bug fixes. For a complete list of changes, see the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/main/CHANGELOG.rst#181-2022-09-01).

Reviewer: ```suggestion
- Upgrading `astronomer-providers` to 1.8.1, which includes various bug fixes. For a complete list of changes, see the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/main/CHANGELOG.rst#181-2022-09-01).
```

Author: Not sure why we want to use gerunds here, when the action is past tense?

Reviewer: @jwitz in a previous edit I recommended changing _Set_ to _Setting_ . If you implemented this change, the list would no longer be parallel. For this reason I recommended changing _Upgrade_ to _Upgrading_. It still works as an improvement or an upgrade can be seen as something that is occurring now.

> - Upgraded `openlineage-airflow` to 0.13.0, which includes fixes for Spark integrations. See the [Astronomer Providers changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md#0141---2022-09-07).

Reviewer: ```suggestion
- Upgrading `openlineage-airflow` to 0.13.0, which includes fixes for Spark integrations. See the [Astronomer Providers changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md#0141---2022-09-07).
```

> - Set `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT=600` by default. 

Reviewer: ```suggestion
- Set the default value for the `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT` variable. to 600 
```

> - Backported several changes from Airflow 2.4.0:

Reviewer 2: Let's make this an H3? As we do here: https://docs.astronomer.io/astro/runtime-release-notes#backported-fixes-from-apache-airflow

(We could specify `Backported fixes from Airflow 2.4`)

    - Properly build URL to retrieve logs independently from system [#26337](https://github.com/apache/airflow/pull/26337)
    - Fix proper joining of the path for logs retrieved from celery workers [#26493](https://github.com/apache/airflow/pull/26493)

>     - Fix UI redirect [#26409](https://github.com/apache/airflow/pull/26409)

Reviewer: ```suggestion
    - Fix UI redirect [#26409](https://github.com/apache/airflow/pull/26409)
```
Do we need to identify the UI we're referencing?

Author: For backported Airflow fixes, I think we should use the same description as the PR we're referencing

Reviewer 2: I actually disagree here -- the Airflow PR descriptions are historically _bad_ so I'm ok giving the user more clarity. This means a little bit more work for us to go interpret a PR, but even a quick review and slight improvement is good.

@jwitz Can you ask @jedcunningham for his opinion and have him confirm what this bug fix actually is?

Reviewer 3: This one was intentionally vague in OSS, as it fixes a CVE: https://lists.apache.org/thread/cn098dcp5x3c402xrb06p3l7nz5goffm

Author: My issue with writing our own notes for each backported fix is that these fixes can get pretty technically nuanced to the point that a quick review might end up misinterpreting the change rather than illuminating it. And given that we're usually on a tight schedule to get these notes published, I don't think it's possible to investigate the changes each time. For people interested in this level of detail, I think the commit number / link is the most important detail to include
